### ISSUE TITLE
Creating product error with startdate

### ISSUE THREAD

**Issue by alexvandervegt** [magento/magento2#1300]
When creating products with php:

\Magento\Catalog\Api\Data\ProductInterface on save function

I get this error since the last update to beta10:

main.CRITICAL: Class Magento\Catalog\Model\Product\Attribute\Backend\Startdate does not exist [] []

**Comment by alexvandervegt** [comment]
Needed to update eav_attribute table, this was not handled with:

bin/magento setup:upgrade

**Comment by mazhalai** [comment]
@alexvandervegt Thank you for reporting this issue. Where did you get this error? Can you be more specific about 
> When creating products with php:
Are you using a custom script?

**Comment by alexvandervegt** [comment]
Hi,

This was in a custom script for adding products automaticly.

It was solved after editing the eav_attribute table in the database. In the last update the:

Magento\Catalog\Model\Product\Attribute\Backend\Startdate

was moved to:

Magento\Catalog\Model\Attribute\Backend\Startdate

bin/magento setup:upgrade did not make this edit in an existing database so I needed to manually edit the table.
